util.py

```diff
@@ -205,4 +195,36 @@ def clean(self):
         """@brief Call when done using switch."""
         self._edge_callback.cancel()
         self._debounce_timer.cancel()
-        self._debounce_timer.join()
\ No newline at end of file
+        self._debounce_timer.join()
+
+class ConfigManager:
+    _lock = Lock()
```

Comment:
Can't we align to web-server? Does it even work as intended that way?
`LOCK_PATH = "/etc/mini-air-quality/envs.lock"`
`with open(LOCK_PATH, 'w') as lock:`
See:
https://github.com/Mini-air-quality-station/web-server/blob/master/html/scripts/save_new_env_val.py